buildenv/buildenv.nix

```diff
@@ -0,0 +1,278 @@
+{
+  activationScripts_storePath ? "@activationScripts@",
+  defaultEnvrc_storePath ? "@defaultEnvrc@",
+  coreutils_storePath ? "@coreutils@",
+  floxBuildenv_storePath ? "@out@",
+  manifest,
```

Comment:
@ysndr any thoughts on calling it `manifest_lock` or `manifestLock`?

Reply:
Ok either way, i just don't want to start guessing whether i pass a lockfile or a manifest because the argument is called manifest but expected to be a lockfile.

Reply:
Thanks - I'll go for `manifestLockfile` to be consistent with the use of `manifestLock` for the JSON-parsed representation below.  👍

Reply:
Fixed with commit 5e009140872270786047c71596587a3e61ba72a3 - resolving conversation